.github/workflows/build.yaml

```diff
@@ -64,6 +65,7 @@ jobs:
         run: nix-build -A outputs.checks.x86_64-linux.${{ steps.params.outputs.attr }}-mysql
 
       - name: Build Xdebug extension
+        if: ${{ steps.params.outputs.major < 8 && steps.params.outputs.minor < 2 }}
```

Comment:
Please also skip it in `checks.nix` so that `nix flake check` can pass.

Reply:
Done, I hope it's ok.